src/drivers/basic/watchdog.h

```diff
@@ -54,7 +54,10 @@ int watchdog_module_init(void);
  * Start the watchdog with given timeout. Use `watchdog_kick()` to
  * periodically restart the timer.
  *
- * @param[in] timeout Watchdog timeout in milliseconds.
+ * @param[in] timeout Watchdog timeout in milliseconds. This value is usually
+ *                    adjusted to a next longer limit in milliseconds because
+ *                    it depends on a hardware prescaler register. Further
+ *                    info in your microcontroller reference manual.
```

Comment:
Nice, but does the timeout depend on a prescaler register in all MCUs? I think it should be rephrased into a more general description.